packages/libzipperposition/libzipperposition.1.5.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "simon.cruanes.2007@m4x.org"
+homepage: "https://github.com/sneeuwballen/zipperposition"
+synopsis: "Library for Zipperposition"
+build: [
+  ["dune" "build" "-p" name]
+  ["dune" "build" "@doc" "-p" name] {with-doc}
+]
+depends: [
+  "base-bytes"
+  "base-unix"
+  "zarith"
+  "logtk" { = version }
+  "containers" { >= "1.0" }
+  "iter" { >= "1.2" }
+  "dune" { build }
```

Comment:
```suggestion
  "dune" { >= "1.1" }
```
Dune should not be tagged as a build dependency and according to the `dune-project` file this package requires at least dune 1.1